### ISSUE TITLE
Cannot get mouse events when react-map-gl map is wrapped by Deckgl

### ISSUE THREAD

**Issue by Harrisandwich** [uber/react-map-gl#655]
I understand this is probably by design, but I have a special case where I would like to be able to get around the Deckgl component that is wrapped around the ReactMapGL component in order to access click events on that map. 

Currently having issues with the DeckGL experimental [TileLayer](https://github.com/uber/deck.gl/tree/master/modules/experimental-layers/src/tile-layer), so I was going to go back to the old mapbox vector layer. The layer adds fine using `map.addLayer(layer)` but all interactivity to that layer is being blocked by either deck.gl or react-map-gl. 

Is there any way I can interact with this vector layer? Perhaps even using deck or react-map-gl?

**Comment by Pessimistress** [comment]
Duplicate of https://github.com/uber/deck.gl/issues/2412
